benches/ycsb/install.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+git clone https://github.com/tarantool/YCSB.git "$PWD"
+mvn clean package
```

Comment:
1. ycsb installation takes too much time. Can we pack files of installed benchmark and use it for measurements? It will reduce preparation time significantly.
2. you are always clone current version of repository, consider using snapshots with fixed commit it. The same for `git clone` for other benchmarks.